version-mapping.json

```diff
@@ -1,10 +1,10 @@
 {
   "4.6": {
-    "index_image": "registry-proxy.engineering.redhat.com/rh-osbs/iib:12345",
-    "bundle_version": "v2.5.0-66"
+    "index_image": "registry-proxy.engineering.redhat.com/rh-osbs/iib:33856",
+    "bundle_version": "v2.5.3-23"
   },
   "4.7": {
-    "index_image": "registry-proxy.engineering.redhat.com/rh-osbs/iib:31338",
-    "bundle_version": "v2.6.0-355"
+    "index_image": "registry-proxy.engineering.redhat.com/rh-osbs/iib:33857",
+    "bundle_version": "v2.5.3-23"
```

Comment:
Shouldn't this point to a 2.6 image somehow?

Reply:
yes, i'll fix it to catch only 2.6 updates.